js/src/lib/interfaces/Libraries.ts

```diff
@@ -16,6 +16,7 @@ export enum ModelLibrary {
 	"pyannote-audio"         = "pyannote.audio",
 	"sentence-transformers"  = "Sentence Transformers",
 	"sklearn"                = "Scikit-learn",
+	"skops"                  = "SKOPS",
```

Comment:
I think we've written `skops` all small letters everywhere. Otherwise, I'm not sure if we want to add it here: https://github.com/huggingface/moon-landing/issues/3757

Reply:
I opened the pr before the discussion, I will close it ;)